bedrock/firefox/templates/firefox/new/scene2.html

```diff
@@ -38,9 +38,17 @@ <h1>{{ _('Let’s do this!') }}</h1>
 <p class="help">
   {# fallback_url is replaced by the platform download link via JS, but if
       something fails the user should still get a link to a working download path. #}
-  {% trans id='direct-download-link', fallback_url=url('firefox.all') %}
-    Your download should begin automatically. If it doesn’t, <a id="{{ id }}" href="{{ fallback_url }}">click here</a>.
-  {% endtrans %}
+
+  {% set id, fallback_url = 'direct-download-link', url('firefox.all') %}
+  {% if l10n_has_tag('firefox_new_try_again_link') %}
```

Comment:
Should this have a date or a bug number in it?

Reply:
I don't believe we have any real conventions for formatting l10n tags, aside from using underscores to separate words. The only thing that matters technically, is that the tag name is unique in the lang file. I don't believe it's really something that localizers would use for reference. Either way, adding a date is a good extra measure, so I'll do that here 👍